### PR TITLE
Non-demo genes overview and change its stats when changing the coverage level

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- New genes overview (non-demo) page
- Try displaying different overview stats by changing the coverage completeness in these links:

image

**How to prepare for test**:
- Deploy on stage 

### How to test:
- Go to demo page: https://chanjo2-stage.scilifelab.se/overview/demo
- Change coverage completeness level

### Expected outcome:
- [x] The page stats should update

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
This is also ready for review now, when you have some time @henrikstranneheim, thanks! 🙏🏻

File: src/chanjo2/endpoints/overview.py [hunk 1]
```diff
@@ -38,3 +38,23 @@ async def demo_overview(request: Request, db: Session = Depends(get_session)):
             "incomplete_coverage_rows": overview_content["incomplete_coverage_rows"],
         },
     )
+
+
+@router.post("/overview", response_class=HTMLResponse)
```
Comment (henrikstranneheim) [inline src/chanjo2/endpoints/overview.py:43]:
```suggestion
@router.post("/gene_coverage_overview", response_class=HTMLResponse)
```
?

**Review by henrikstranneheim — APPROVED** [review]
Nice 👍

File: src/chanjo2/endpoints/overview.py (hunk 1 above)
Comment (northwestwitch) [reply src/chanjo2/endpoints/overview.py:43]: geneS in case, since gene_overview is another endpoint that is missing

Comment (northwestwitch) [reply src/chanjo2/endpoints/overview.py:43]: I think it's also fine leaving it like it is and creating the new endpoint with name "gene_overview"

**Pull request merged**